paket.dependencies

```diff
@@ -11,4 +11,6 @@ nuget FSharp.Core
 github forki/FsUnit FsUnit.fs
 github fsharp/FAKE modules/Octokit/Octokit.fsx
 github fsharp/FAKE src/app/FakeLib/Globbing/Globbing.fs
-github fsprojects/Chessie src/Chessie/ErrorHandling.fs
\ No newline at end of file
+github fsprojects/Chessie src/Chessie/ErrorHandling.fs
+nuget FsCheck
```

Comment:
did you add this manually? If you used paket add I hope it would put it to the other nuget statements (in alphabetical order)